lib/balance-tx/cardano-balance-tx.cabal

```diff
@@ -40,7 +40,33 @@ library
   build-depends:
     , base
     , cardano-coin-selection
+    , cardano-slotting
+    , ouroboros-consensus
     , cardano-wallet-primitive
+    , cardano-strict-containers
+    , cardano-ledger-shelley
+    , cardano-ledger-core
+    , cardano-api
+    , cardano-ledger-conway
+    , cardano-ledger-binary
+    , cardano-ledger-api
+    , text
+    , lens
+    , serialise
+    , cborg
+    , monoid-subclasses
+    , pretty-simple
+    , random
+    , std-gen-seed
+    , cardano-ledger-mary
+    , cardano-ledger-babbage
+    , cardano-ledger-alonzo
+    , cardano-ledger-allegra
+    , cardano-crypto-class
+    , ouroboros-consensus-cardano
+    , int-cast
+    , cardano-addresses
+    , bytestring
```

Comment:
Is it possible to sort these dependencies into alphabetical order? 🙏🏻 
(It's easier to spot duplicates that way)